web/themes/custom/cloudy/templates/search/search-api-page.html.twig

```diff
@@ -11,6 +11,7 @@
   {% else %}
     {{ no_results_found }}
   {% endif %}
+  To improve search results, be specific. Use quotes to search for exact text.
```

Comment:
@ccdatapdx , I altered the text a bit to make it work a bit better with the results statement. I also moved it below the if/else clause so that it is only printed once. (Don't repeat yourself if you don't have to 😉 )